ActionRPG/Tile.h

```diff
@@ -11,6 +11,9 @@ class Tile
 	bool m_isTopWallDrawn;
 	sf::FloatRect m_bounds;
 
+	// Flag for if the tile is colliding with a GameObject
+	bool m_isCollidingWithGameObject;
```

Comment:
A-ha! This is my key! I forgot about this! maybe tiles should also just have an array of  `GameObjects` that they collide with, and I loop through to check this array after I check the tile collisions. I think this is the approach I want to take!

Reply:
running into a circular dependency if I attempt to do this though, which is breaking the build I think